src/main/java/com/cashcard/CashCardController.java

```diff
@@ -23,6 +23,14 @@ private ResponseEntity<CashCard> findById(@PathVariable Long requestedId) {
         } else {
             return ResponseEntity.notFound().build();
         }
+    }
+
+    @PostMapping
+    private ResponseEntity<Void> createCashCard(@RequestBody CashCard newCashCardRequest, UriComponentsBuilder ucb) {
```

Comment:
you are not returning anything, why not PUT?

Reply:
In the response, I'm returning the location of the created cash card in the header and status code.
No change was done.